solarforecastarbiter/reports/templates/pdf/macros.j2

```diff
@@ -236,7 +236,8 @@
 \VAR{data_validation_post_text() | html_to_tex}
 %- if report.report_parameters.filters
 \VAR{validation_metadata_table(report.report_parameters.filters)}
-\VAR{validation_results_table(report.raw_report.processed_forecasts_observations, report.raw_report.processed_forecasts_observations | unique_flags_filter)}
+\VAR{validation_results_table(report.raw_report.processed_forecasts_observations, report.raw_report.processed_forecasts_observations | unique_flags_filter(true))}
+\VAR{validation_results_table(report.raw_report.processed_forecasts_observations, report.raw_report.processed_forecasts_observations | unique_flags_filter(false))}
```

Comment:
The problem w/ the tex is that unique_flags is empty and there isn't logic to make empty columns 
![image](https://user-images.githubusercontent.com/4625457/97601933-f3899b00-19c7-11eb-9b39-09ed287efba8.png)

Reply:
hmm maybe that's not it?

Reply:
best solution is to not display the table at all and instead show some warning that you might consider recomputing the report to get the after validation counts for both pdf and html

Reply:
This is what the html looks like. Existing reports will look like this until they are recomputed: <img width="1157" alt="Screen Shot 2020-10-29 at 10 00 28 AM" src="https://user-images.githubusercontent.com/4383303/97606973-9d1f5b00-19cd-11eb-8db4-e2cdec4f66b8.png">

Recomputed reports will always have both kinds of flags because ISNAN always exists for both before and after.

For html, I'm not sure if it's worth the effort to conditionally display the table, nor do I have a good idea of how to explain the situation in the report text. Maybe a line "This table will be empty if viewing a report computed with Solar Forecast Arbiter version 1.0rc3 or earlier."? I dunno. We're still in pre-1.0 so I don't feel that strongly about it.

For pdf, I think conditionally displaying the table makes sense. The slight difference in code structure actually makes that easier in that macro.

Reply:
fine with me. could also check the length of the unique flags at the start of the macro and if 0, `%- set unique_flags = ['']`